pettingzoo/magent/magent_env.py

```diff
@@ -127,6 +145,40 @@ def _all_dones(self, step_done=False):
         ret_agents = set(self.agents)
         return {agent: bool(done) for agent, done in zip(self.possible_agents, dones) if agent in ret_agents}
 
+    def state(self):
+        '''
+        Returns an observation of the global environment
+        '''
+        state = np.copy(self.base_state)
+
+        for handle in self._all_handles:
+            view, features = self.env.get_observation(handle)
+
+            pos = self.env.get_pos(handle)
+            pos_x, pos_y = zip(*pos)
+            state[pos_x, pos_y, 1 + handle.value * 2] = 1
+            state[pos_x, pos_y, 2 + handle.value * 2] = np.copy(view[:, view.shape[1] // 2, view.shape[2] // 2, 2])
```

Comment:
No need for a copy here

Reply:
I'll remove this after making a decision for the previous comment :)